src/test/java/com/playground/playground/controller/DataAttributesControllerTest.java

```diff
@@ -0,0 +1,126 @@
+package com.playground.playground.controller;
+
+import com.playground.playground.interface_adapter.controller.DataAttributesController;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.control.Slider;
+import javafx.scene.input.MouseEvent;
+import javafx.stage.Stage;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.testfx.framework.junit5.ApplicationTest;
+
+import static org.junit.Assert.assertEquals;
+
+class DataAttributesControllerTest extends ApplicationTest {
```

Comment:
No class javadocs

Reply:
Wasn't provided for other tests, I'll change that upon merge.